h2o-automl/src/main/java/ai/h2o/automl/IAlgo.java

```diff
@@ -0,0 +1,9 @@
+package ai.h2o.automl;
+
+import java.io.Serializable;
+
+public interface IAlgo extends Serializable {
+  String name();
+  default String urlName() { return name().toLowerCase(); }
+  default boolean enabled() { return true; }
+}
```

Comment:
introduced this interface for 2 purposes:
1. being able to test `ModelingStep` internals, using a very fast Dummy algo.
2. offer the possibility to plug any algo to AutoML, including those not defined in the Algo enum.